.github/workflows/pull-request.yml

```
@@ -544,3 +544,26 @@ jobs:
         run: |
           source venv/bin/activate
           python .github/workflows/scripts/check_docstrings.py --directories .github
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)